toolkit/src/styles/components/list-item.scss

```diff
@@ -0,0 +1,54 @@
+/**
+* Copyright (c) HashiCorp, Inc.
+*/
+
+@import '@hashicorp/design-system-components';
+
+.cut-list-item {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 0;
```

Comment:
this is assuming that the parent is _always_ a flexbox; is this the case?

Reply:
which leads me to another question: since this is the list-**item**, there should also be a **list** parent component?